modules/serverless-sinkbinding-kn.adoc

```diff
@@ -87,13 +84,12 @@ spec:
                     fieldRef:
                       fieldPath: metadata.namespace
 ----
-
 +
 [IMPORTANT]
 ====
-To use SinkBinding, you must manually add a `bindings.knative.dev/include=true` label to your Knative resources.
+To use sink binding, you must manually add a `bindings.knative.dev/include=true` label to your Knative resources.
```

Comment:
It is also possible to just label the namespace of the cronjob: https://knative.dev/docs/eventing/sources/sinkbinding/ 

Maybe it is mentioned somewhere else, can't see the entire context in Github PR.

Reply:
It's mentioned right below this in the same note specifically for the CronJob :slightly_smiling_face: there should be a link for the docs preview generated in the PR -> Conversation tab for future reference.